public_html/wp-content/mu-plugins/blocks/source/components/block-content/index.js

```diff
@@ -26,7 +26,7 @@ import './style.scss';
  */
 export function BlockNoContent( { loading } ) {
 	return (
-		<div className="wordcamp-block-content-none">
+		<div className="wordcamp-block_posts has-no-content">
```

Comment:
`MUST` - Looks like this was intended to be a `__`

Reply:
Yup, thanks for the catch. Fixed in https://github.com/WordPress/wordcamp.org/pull/123/commits/92ab54753af3ccb3e8248f03065b2e340521a261